README.md

```diff
@@ -25,10 +25,14 @@ To make properties file:
 
     $ cp config/application.example.properties config/application.properties  # Update properties as necessary (the defaults are fine for running locally)
 
+In particular, make sure the `server.port` value is the same as the port from `FORMPLAYER_URL` in commcarehq's `localsettings.py`.
+
 Make sure you have the formplayer database created. You will be asked to provide a password after running this command; assuming you are running formplayer locally, you should use the password for the postgres user associated with the locally-running instance of commcare hq (which can be found in the `DATABASES` section of your `localsettings.py` file).
 
     $ createdb formplayer -U commcarehq -h localhost  # Update connection info as necessary (the defaults are fine for running locally)
 
+If you are running postgres in Docker, you may need to run this in the docker shell, using `./scripts/docker bash postgres` from the commcarehq repository.
```

Comment:
really? I think you should be able to access postgres without having to run this command directly in the container

Reply:
I never installed postgres on my machine, so `createdb` only existed in the container

Reply:
ah, makes sense